packages/ingest/test/test-SftpProviderClient.js

```diff
@@ -177,7 +177,7 @@ test('SftpProviderClient.list filters listed objects with path', async (t) => {
   t.is(list[0].name, 'MOD09GQ_1granule_v3.PDR');
 });
 
-test('Download remote file to s3 with correct content-type', async (t) => {
+test.serial('Download remote file to s3 with correct content-type', async (t) => {
```

Comment:
why do these tests need to be serialized? they're using randomized keys, what's keeping them tied to each other's resources?

Reply:
They don't.  I hope to reduce the unit test errors from SFTP server, like this one:
```

16-Oct-2024 18:57:17 | @cumulus/ingest:   Uncaught exception in test/test-SftpProviderClient.js
-- | --
16-Oct-2024 18:57:17 | @cumulus/ingest:   Error: No response from server
16-Oct-2024 18:57:17 | @cumulus/ingest:   › cleanupRequests (/cumulus/packages/sftp-client/node_modules/ssh2/lib/protocol/SFTP.js:2730:15)
16-Oct-2024 18:57:17 | @cumulus/ingest:   › SFTP.push (/cumulus/packages/sftp-client/node_modules/ssh2/lib/protocol/SFTP.js:191:7)
16-Oct-2024 18:57:17 | @cumulus/ingest:   › onCHANNEL_CLOSE (/cumulus/packages/sftp-client/node_modules/ssh2/lib/utils.js:50:13)
16-Oct-2024 18:57:17 | @cumulus/ingest:   › ChannelManager.cleanup (/cumulus/packages/sftp-client/node_modules/ssh2/lib/utils.js:200:7)
16-Oct-2024 18:57:17 | @cumulus/ingest:   › Socket.<anonymous> (/cumulus/packages/sftp-client/node_modules/ssh2/lib/client.js:829:********)
16-Oct-2024 18:57:17 | @cumulus/ingest:   ✘ test/test-SftpProviderClient.js exited with a non-zero exit code: 1
```

Similar errors in master with serial.
https://ci.earthdata.nasa.gov/browse/CUM-CBA-RUT-2078/log
```
@cumulus/sync-granule:   Uncaught exception in tests/sync_granule_test.js
--
08-Oct-2024 22:25:53 | @cumulus/sync-granule:   Error: No response from server
08-Oct-2024 22:25:53 | @cumulus/sync-granule:   › cleanupRequests (/cumulus/packages/sftp-client/node_modules/ssh2/lib/protocol/SFTP.js:2730:15)
08-Oct-2024 22:25:53 | @cumulus/sync-granule:   › SFTP.push (/cumulus/packages/sftp-client/node_modules/ssh2/lib/protocol/SFTP.js:191:7)
08-Oct-2024 22:25:53 | @cumulus/sync-granule:   › onCHANNEL_CLOSE (/cumulus/packages/sftp-client/node_modules/ssh2/lib/utils.js:50:13)
08-Oct-2024 22:25:53 | @cumulus/sync-granule:   › ChannelManager.cleanup (/cumulus/packages/sftp-client/node_modules/ssh2/lib/utils.js:200:7)
08-Oct-2024 22:25:53 | @cumulus/sync-granule:   › Socket.<anonymous> (/cumulus/packages/sftp-client/node_modules/ssh2/lib/client.js:829:********)
08-Oct-2024 22:25:53 | @cumulus/sync-granule: {"level":"debug","message":"returning {\"size\":1098034,\"bucket\":\"037af5c553594516825ebe6fc77489ea13359719\",\"key\":\"file-staging/lpdaac-cumulus-phaseIII/MOD09GQ___6/MOD09GQ.A2017224.h27v08.006.2017227165029.hdf\",\"source\":\"/granules/MOD09GQ.A2017224.h27v08.006.2017227165029.hdf\",\"fileName\":\"MOD09GQ.A2017224.h27v08.006.2017227165029.hdf\",\"type\":\"data\",\"checksumType\":\"CKSUM\",\"checksum\":\"143571********44\"}","sender":"unknown","timestamp":"2024-10-09T02:25:52.567Z"}
08-Oct-2024 22:25:53 | @cumulus/sync-granule: {"level":"debug","message":"finished ingest()","sender":"unknown","timestamp":"2024-10-09T02:25:52.567Z"}
08-Oct-2024 22:25:53 | @cumulus/sync-granule: {"level":"debug","message":"SyncGranule Complete. Returning output: {\"granules\":[{\"granuleId\":\"MOD09GQ.A2017224.h27v08.006.2017227165029\",\"dataType\":\"MOD09GQ\",\"version\":\"6\",\"provider\":\"MODAPS\",\"files\":[{\"size\":1098034,\"bucket\":\"037af5c553594516825ebe6fc77489ea13359719\",\"key\":\"file-staging/lpdaac-cumulus-phaseIII/MOD09GQ___6/MOD09GQ.A2017224.h27v08.006.2017227165029.hdf\",\"source\":\"/granules/MOD09GQ.A2017224.h27v08.006.2017227165029.hdf\",\"fileName\":\"MOD09GQ.A2017224.h27v08.006.2017227165029.hdf\",\"type\":\"data\",\"checksumType\":\"CKSUM\",\"checksum\":\"143571********44\"}],\"sync_granule_duration\":8849,\"createdAt\":1728440743718}],\"granuleDuplicates\":{},\"process\":\"modis\"}","sender":"unknown","timestamp":"2024-10-09T02:25:52.571Z"}
08-Oct-2024 22:25:53 | @cumulus/sync-granule:   ✘ tests/sync_granule_test.js exited with a non-zero exit code: 1
```

Reply:
awesome, this is the same error I'm trying to track down separately, but I guess it's good to know one piece: that it isn't solved by serialization